core/src/from_meta.rs

```diff
@@ -795,8 +813,12 @@ mod tests {
 
     #[track_caller]
     fn fm<T: FromMeta>(tokens: TokenStream) -> T {
+        try_fm(tokens).expect("Tests should pass valid input")
+    }
+
+    #[track_caller]
+    fn try_fm<T: FromMeta>(tokens: TokenStream) -> Result<T> {
```

Comment:
Rather than this, could you use `#[should_panic]` on the failing test?

Reply:
Sure thing, I just pushed this change in 47f9a4c230001f42178d9c6d89e620268abe90ed